src/bit-u.md

```diff
@@ -513,9 +513,11 @@ a.quote-link:hover{
     <p>Or as we host incredible speakers with Q&A at the monthly ChicagoJS Meetups.</p>
     <a class="button" href="https://www.meetup.com/js-chi/events/"><img src="./static/img/calendar.svg" height="20"> ChicagoJS Meetup Schedule</a>
     <!-- Community calendar temporarily removed until the calendar is consistently updated again. -->
-    <!-- <p>Every two weeks, Bitovi hosts a live training. Subscribe to Bitovi's community calendar to be part of the next one!</p> 
-    <a class="button" href="https://calendar.google.com/calendar/b/1?cid=anVwaXRlcmpzLmNvbV9nMjd2Y2szNm5pZmJucXJna2N0a29hbnFiNEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t"><img src="./static/img/calendar.svg" height="20"> Subscribe to our calendar</a> -->
-  <!-- <calendar-events api-key="AIzaSyBsNpdGbkTsqn1BCSPQrjO9OaMySjK5Sns" calendar-id="jupiterjs.com_g27vck36nifbnqrgkctkoanqb4@group.calendar.google.com" event-count="3" class="courses">
+    <p>Join our community calendar for even more upcoming events.</p> 
+    <a class="button" href="https://calendar.google.com/calendar/b/1?cid=anVwaXRlcmpzLmNvbV9nMjd2Y2szNm5pZmJucXJna2N0a29hbnFiNEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t"><img src="./static/img/calendar.svg" height="20"> Subscribe to our calendar</a>
+    <!-- when running locally, must use http://localhost:5500/academy/ instead of ip address to see calendar events locally -->
```

Comment:
Do you mean that when you're running the academy locally, your browser has to be on localhost? If so, put this in the readme instead of here. =]

Reply:
Yes. Moved it to the readme & attempted to add clarification.